vendor/react.js

```diff
@@ -1,3 +1,8 @@
-import react from '../react.js';
+import { returnOrThrow } from '../helper.js';
 
-export default react;
+/**
+ * React object used by Camunda Modeler. Use it to create UI extension.
+ *
+ * @type {import('react')}
+ */
+export default returnOrThrow(() => window.react, '3.4');
```

Comment:
If we are exporting via vendor shoudn't it be exposed by window.vendor?.react (ref. https://github.com/camunda/camunda-modeler/pull/4860#issuecomment-2711429915)

```suggestion
export default returnOrThrow(() => window.vendor?.react, '3.4');
```

so in modeler, when it calls for react package from helper, it will look into vendor object to see if it has react package setup.

Reply:
No, we cannot do this, as it would be a major (Camunda Modeler) platform breaking change.

Of course, we could start to export `react` from `window.vendor` moving forward, but this will break _any_ existing plug-in.